frontstage/views/sign_in/sign_in.py

```diff
@@ -70,6 +71,8 @@ def login():  # noqa: C901
                 bound_logger.info("Bad credentials provided")
             elif UNKNOWN_ACCOUNT_ERROR in error_message:
                 bound_logger.info("User account does not exist in auth service")
+            elif USER_ACCOUNT_DELETED in error_message:
+                bound_logger.info("User account is marked for deletion")
```

Comment:
Should this be something like:
```python
bound_logger.info("User account is marked for deletion.  Contact us if you believe this has been marked in error")
```
or something like that?  Might need to ask Gareth what he thinks.  I could also be wrong so ignore this if you want ^^

Reply:
👍

Reply:
Although its just a logger

Reply:
checked with Gareth. He is happy as it is

Reply:
I'm dumb, it is just a logger.  Sorry >_<